files/settings-default.cfg

```diff
@@ -1,244 +1,709 @@
+# the value of this setting (since the cursor/crosshair is always
+# centered in the OpenMW window).  However, in GUI mode, this setting
+# determines the behavior when the cursor is moved outside the OpenMW
+# window.  If true, the cursor movement stops at the edge of the
+# window preventing access to other applications.  If false, the
+# cursor is allowed to move freely on the desktop.
+#
+# This setting does not apply to the screen where escape has been
+# pressed, where the cursor is never captured.  Regardless of this
+# setting "Alt-Tab" or some other operating system dependent key
+# sequence can be used to allow the operating system to regain control
+# of the mouse cursor.  This setting interacts with the "minimize on
+# focus loss" setting by affecting what counts as a focus loss.
+# Specifically on a two-screen configuration it may be more convenient
+# to access the second screen with setting disabled.
+grab cursor = true
+
+# Invert the vertical axis while in "look mode".  If this setting is
+# true, moving the mouse away from the player will look down, while
+# moving it towards the player will look up.  This setting does not
+# affect cursor movement in GUI mode.
+invert y axis = false
+
+# This boolean setting causes the behavior of the sneak key (Ctrl by
+# default) to toggle sneaking on and off rather than requiring the key
+# to be held while sneaking.  Players that spend significant time
+# sneaking may find the character easier to control with this option
+# enabled.
+toggle sneak = false
+
+# This setting continues to be loaded and saved, but has no known
+# effect.  Presumably it and a related but also removed option named
+# "ui y sensitivity" used to control mouse sensitivity while in GUI
+# mode.  The default value is 1.0.
+ui sensitivity = 1.0
 
 [Map]
-# Adjusts the scale of the global map
+
+# It is not currently possible to control how many adjacent cells are
+# displayed in the map.  It appears that this is hardcoded to one
+# adjacent cell (3x3) in the code.  These settings control the canvas
+# and resolution sizes, and therefore the amount of panning required
+# to see the entire map, and the level of detail visible.
+
+# This integer setting adjusts the scale of the world map in the GUI
+# mode map display. The value is the width in pixels of each cell in
+# the map, so larger values result in larger more detailed world maps,
+# while smaller values result in smaller less detailed world maps.
+# However, the native resolution of the map source material appears to
+# be 9 pixels per unexplored cell and approximately 18 pixels per
+# explored cell, so values larger than 36 don't produce much
+# additional detail.  Similarly, the size of place markers is
+# currently fixed at 12 pixels, so values smaller than this result in
+# overlapping place markers.  Values from 12 to 36 are recommended.
+# For reference, Vvardenfell is approximately 41x36 cells.
```

Comment:
This setting needs to be marked with a warning that it affects save files. The currently explored global map overlay is stored in the save file as an image. When you increase the map size, the overlay of pre-existing save games will be up-scaled on load (thus become blurry). Note that when you visit a cell again the overlay for that cell gets re-generated, so you can "fix" the problem by revisiting all cells you've already visited.